super/S_Login/bot_new.py

```
@@ -218,28 +222,7 @@ def log_to_wiki_1(self, do=False) -> str:
-            files (dict?): A dictionary of files to upload with the request. Defaults to None.
-            timeout (int?): The timeout for the request in seconds. Defaults to 30.
-
-        Returns:
-            any or None: The response object from the POST request, or None if the
-                request fails.
-
-        Raises:
-            Exception: If the user table is not ready when attempting to send the request.
-            requests.exceptions.ReadTimeout: If the request times out.
-        """
+    def raw_request(self, params, files=None, timeout=30) -> any or None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)